examples/.NET/README.md

```diff
@@ -0,0 +1,23 @@
+# GC Forms API client example in .NET
+
+## Prerequisites
+
+- Download and install .NET version 8.0
+  - (with Microsoft installer) https://dotnet.microsoft.com/en-us/download
+  - (with Brew) `brew install --cask dotnet`
+
+To make sure .NET is properly installed, try to run the following command `dotnet --version`.
+
+## How to setup application
+
+- Put the private key file you have generated through the GCForms web application in this folder next to the `Program.cs` file. Make sure the file name ends with `_private_api_key.json`
+- In the `Program.cs` file set the following variables:
+  - `IDENTITY_PROVIDER_URL`: This is the URL of the identity provider (IDP)
+  - `PROJECT_IDENTIFIER`: This is the GCForms project identifier in the IDP
+  - `GCFORMS_API_URL`: This is the URL of the GCForms API
```

Comment:
Do you know if there's a way to set these via a `.env` file?  Also what would you think of providing example values for the two URLs and just setting them to prod?

Reply:
That is something we could do. I went with this solution because, I thought it would be easier for developers to start using the example if they only had to edit some strings.

Also I think the idea for developers looking at this example, would be to quickly import parts of the code without having to think about what else needs to be imported so that it works right away in their own project.

Reply:
It's not sensitive at all.  It'll need to be in our public documentation so that clients can connect to us.  It's needed so that the client give GCForms permission to use their JWT token.

Reply:
This is what I thought! Nice then I will just set the variables to target the production environment.